libs/blocks/fragment/fragment.js

```diff
@@ -63,6 +63,13 @@ export default async function init(a) {
   let relHref = localizeLink(a.href);
   let inline = false;
 
+  if (a.parentElement && a.parentElement.nodeName === 'P') {
```

Comment:
```suggestion
  if (a.parentElement?.nodeName === 'P') {
```

Reply:
Nice! Don't know how I missed this 🤦‍♂️ Made the change 👍